WordPress/Classes/Services/CommentServiceRemoteCoreRESTAPI.swift

```diff
@@ -0,0 +1,323 @@
+import Foundation
+import WordPressCore
+import WordPressAPI
+import WordPressAPIInternal
+import WordPressKit
+import WordPressShared
+import WordPressData
+
+final class CommentServiceRemoteCoreRESTAPI: NSObject, CommentServiceRemote {
+    private let client: WordPressClient
+
+    init(client: WordPressClient) {
+        self.client = client
+    }
+
+    func getCommentsWithMaximumCount(_ maximumComments: Int, success: (([Any]?) -> Void)?, failure: (((any Error)?) -> Void)?) {
+        getCommentsWithMaximumCount(maximumComments, options: nil, success: success, failure: failure)
+    }
+
+    func getCommentsWithMaximumCount(_ maximumComments: Int, options: [AnyHashable: Any]? = [:], success: (([Any]?) -> Void)?, failure: (((any Error)?) -> Void)?) {
+        Task { @MainActor in
```

Comment:
What happens to the various tasks in here if this class is dealloc'd?

Should we be tracking Task handles so that we can cancel the tasks in `deinit`?